src/pages/components/tabs/style.mdx

```diff
@@ -357,7 +388,7 @@ Tab labels should be set in sentence case, and should not exceed three words.
 
 <div className="image--fixed">
 
-![Structure and spacing measurements for line tabs.](images/tab-style-7.png)
+![Structure and spacing measurements for line tabs.](images/tab-style-13.png)
```

Comment:
- When showing icons specs in magenta in images, we are usually trying to show this as 16px, instead of specifying the rem, since its an svg size. 
- For one image, you only need to specify the px/rem one time.